docs/satellite-clone.md

```diff
@@ -3,12 +3,9 @@
 #### Important Notes ####
 Before cloning:
   - It is recommended that the target server is network isolated. We take steps to disable communication from the target server to existing capsules and hosts, but the only way to ensure this is in an isolated environment.
-  - If your storage is slow (like NFS) and your pulp backup tar file is large (>100 gb), you might see memory errors while untaring pulp data during the playbook run. In this case don't include pulp_data.tar during the cloning process and instead copy over `/var/lib/pulp` from the source server to the target server after the cloning process. For example, from the source server:
-
-     ```console
-        # rsync -az /var/lib/pulp target_server:/var/lib/pulp
-    ```
-    Note that rsync command may take a while to complete to copy large data.
+  - If your storage is slow (like NFS) and your pulp backup tar file is large (>100 gb), you might see memory errors while untaring pulp data during the playbook run. In this case, don't include pulp_data.tar during the cloning process.  After the clone run *without pulp_data.tar*, you have two options:
+    - To recover pulp content after the cloning process, follow these [steps](#cloning-without-pulp_contenttar-and-recover-pulp-content-after-cloning).
```

Comment:
We should specify the pulp_data part here, its still not clear that this step isn't required when you clone w/ pulp_data.tar when you first read this.

Reply:
the previous line states - `After the clone run *without pulp_data.tar*, you have two options:`. do you have recommendations on how you want this line to be?

Reply:
ok I see now, I missed that part, it looks much more obvious when I look at it formatted.

One thing is I'm not crazy about making assumptions and giving users the If this doesn't work, then do this phrasing. Can you have a separate line under the memory error part for the second sentence that says something along the lines of: `If you would like to use a backup without a pulp_data.tar file, you have these two options`. I'm guessing a lot of users will want to do this from the start (regardless of memory issues).